packages/-ember-data/tests/integration/adapter/find-test.js

```diff
@@ -59,17 +63,24 @@ module('integration/adapter/find - Finding Records', function(hooks) {
         },
       })
     );
-
     this.owner.register('serializer:application', JSONAPISerializer.extend());
 
-    run(() => {
-      store.findRecord('person', 1);
-      store.findRecord('person', 1);
-    });
+    const store = this.owner.lookup('service:store');
+
+    store.findRecord('person', 1);
```

Comment:
wherever we are currently allowing numeric ID to be passed in as an argument we should switch to string ID, unless is it an explicit test that "passing by number" works. This extends more generally to "lets use string IDs in tests unless explicitly testing a number", as it will make it that much easier to work to constrain the system in the future.